docs/agents/magpie.rst

```diff
@@ -0,0 +1,121 @@
+.. highlight:: rst
+
+.. _magpie:
+
+===============
+Magpie Agent
+===============
+
+.. image:: ../_static/images/magpie.jpeg
+  :width: 400
+  :alt: Alternative text
+
+The magpie is an incredibly intelligent bird, with a decent ability to mimic
+other bird calls, though not as good as `the superb lyrebird
+<https://www.youtube.com/watch?v=mSB71jNq-yQ>`_. In the context of OCS, the job
+of the Magpie agent is to parse data being passed from the SMuRF Streamer, and
+to translate it into a stream that Lyrebird can interpret and display.
+
+The Magpie agent should be setup and run on the same node that is running
+lyrebird, which must have some sort of graphics card (either integrated or
+dedicated), and must be on the same network as the smurf-server for real-time
+playback.
+
+.. argparse::
+   :filename: ../agents/magpie/magpie_agent.py
+   :func: make_parser
+   :prog: python3 magpie_agent.py
+
+Detector Layouts
+-----------------
+
+Grid Layout
+````````````
+The grid layout is a grid with 4096 elements (maximum number of channels a
+single smurf slot can stream)
+
+.. image:: ../_static/images/lyrebird_grid.jpg
```

Comment:
Seems like we should be able to get better images here... are the graphics not screen-grabbable for some reason?